src/helpers/charts/bar_chart.ts

```diff
@@ -200,7 +199,7 @@ function getBarConfiguration(chart: BarChart, labels: string[]): ChartConfigurat
   const legend: ChartLegendOptions = {
     labels: { fontColor },
   };
-  if ((!chart.labelRange && chart.dataSets.length === 1) || chart.legendPosition === "none") {
+  if ((!chart.labelRange && chart.getDataSets().length === 1) || chart.legendPosition === "none") {
```

Comment:
we can probably give the datasets as argument of the method, to avoit building them 2 times